src/Model/Lumberjack.php

```diff
@@ -100,21 +100,23 @@ public function stageChildren($showAll = false)
         }
 
         $this->owner->extend("augmentStageChildren", $staged, $showAll);
-        $this->excludeSiteTreeClassNames($staged);
+        $staged = $this->excludeSiteTreeClassNames($staged);
         return $staged;
     }
 
     /**
      * Excludes any hidden owner subclasses
```

Comment:
Could you please add a message here to say that it returns a new DataList to what is given in the original arguments?